js/bucket.js

```diff
@@ -156,7 +161,9 @@ function groupBy(list, key) {
  * @return {Object} obj with key for each group.
  */
 export function groupedByInterval(jsonData, fieldName, start, end, interval, groupField) {
-  const grouped = groupField ? groupBy(jsonData, groupField) : { [noGroupsLabel] : jsonData};
+  const grouped = groupField
+    ? groupBy(jsonData, groupField)
+    : { [noGroupsLabel]: jsonData };
 
```

Comment:
I'm not a fan of multiple line ternary operator. Is this configurable?

Reply:
Not from what I can tell. It appears that if the line length exceeds the `printWidth` setting, this is what it does.

Reply:
Maybe if the formatter does this we have our own convention to just convert this to an if-then statement.